src/ApiService/ApiService/Program.cs

```diff
@@ -82,10 +82,12 @@ public static void Main()
             .AddScoped<IReports, Reports>()
             .AddScoped<INotificationOperations, NotificationOperations>()
             .AddScoped<IUserCredentials, UserCredentials>()
+            .AddScoped<ICreds, Creds>()
```

Comment:
I think we can just use Icred as the singleton instead of creating a new entity. Plus the name is more descriptive.

Reply:
HttpClient should stay in Singleton resources ? Or move it to Creds ?